code/defines/obj.dm

```diff
@@ -65,15 +65,16 @@ var/global/list/PDA_Manifest = list()
 /datum/datacore/proc/get_manifest_list()
 	if(PDA_Manifest.len)
 		return
-	var/heads[0]
-	var/sec[0]
-	var/eng[0]
-	var/med[0]
-	var/sci[0]
-	var/car[0]
-	var/civ[0]
-	var/bot[0]
-	var/misc[0]
+	var/list/heads = list()
+	var/list/sec = list()
+	var/list/eng = list()
+	var/list/med = list()
+	var/list/sci = list()
+	var/list/car = list()
+	var/list/pla = list() // Planetside crew: Explorers, Pilots, S&S

```

Comment:
I'd comment these "Planetside" manifest categories out, or simply snip the display portion for this list on the PDA manifest. They don't collapse when they are empty, so we'll eternally have an empty department listed otherwise.